interface/ceed-preconditioning.c

```diff
@@ -1135,8 +1135,8 @@ int CeedOperatorAssemblyDataCreate(Ceed ceed, CeedOperator op, CeedOperatorAssem
       }
     }
   }
-  (*data)->num_eval_mode_out = num_eval_mode_out;
-  (*data)->eval_mode_out     = eval_mode_out;
+  (*data)->num_active_out = num_active_out;
+  (*data)->eval_mode_out  = eval_mode_out;
   CeedCall(CeedBasisReferenceCopy(basis_out, &(*data)->basis_out));
 
```

Comment:
@jeremylt I was thing to change `basis_in/out` to array `*basis_in/out` in `CeedOperatorAssemblyData_private`, then put `CeedBasisReferenceCopy` in the loop so we get the active bases, what do you think?

Reply:
Yeah, I create a loop in `CeedOperatorAssemblyDataDestroy` function to destroy all the active bases.